spec/integration/riak/time_series_spec.rb

```diff
@@ -4,8 +4,8 @@
 describe 'Time Series', test_client: true, integration: true do
   let(:table_name){ 'GeoCheckin' }
 
-  let(:now){ Time.now }
-  let(:fiveMinsAgo){ Time.now - 300 }
+  let(:now){ Time.at(Time.now.to_i) }
```

Comment:
Ruby `Time` objects and `Time.now` have higher precision than Riak TS. By truncating them to seconds on initialization, I avoid having to do complex rounding when checking to see if data returning from Riak has the matching time.